crates/build/src/lib.rs

```diff
@@ -218,8 +225,12 @@ impl BuildResult {
         }
         if let Some(metadata_result) = self.metadata_result.as_ref() {
             let metadata = format!(
-                "  - {} (the contract's metadata)",
-                util::base_name(&metadata_result.dest_metadata).bold()
+                "  - {} (the contract's {})",
+                util::base_name(&metadata_result.dest_metadata).bold(),
+                match metadata_result.spec {
+                    MetadataSpec::Ink => "metadata",
+                    MetadataSpec::Solidity => "Solidity compatible ABI",
```

Comment:
```suggestion
                    MetadataSpec::Solidity => "Solidity compatible metadata",
```

For a consistent naming throughout.

Reply:
I'm not sure about this one, this line is for the `<artifact_name>.abi` file. See image below:
<img width="413" alt="Screenshot 2025-02-07 at 15 43 27" src="https://github.com/user-attachments/assets/528b1d93-40c2-411e-8150-675befe8bc36" />

With this suggested change, this becomes
<img width="441" alt="Screenshot 2025-02-07 at 15 52 36" src="https://github.com/user-attachments/assets/ffc1c88d-e932-477b-ab8a-67cb27b52d9f" />